{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/conftest.py

```diff
@@ -1,12 +1,37 @@
 # pylint: disable=redefined-outer-name
 # flake8: noqa: E800
 import os
-from collections.abc import Generator
+from typing import Generator
 
 import pytest
 from playwright.sync_api import BrowserContext, ConsoleMessage, Error, Page, Playwright
 
 
+def pytest_collection_modifyitems(items: list[pytest.Item]) -> None:
+    """
+    Check if any tests are marked as integration.
```

Comment:
This both checks *and* modifies, right?

I assume this is overriding some built-in Pytest function. Can you add a link to pytest docs or source taht this is overriding?

Reply:
Right, the docstring needs to be reworded. And I'll include a link by the function too, here's it now for your reference: https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.hookspec.pytest_collection_modifyitems